### PR TITLE
Decouple certification request from other configuration

### PR DESCRIPTION
**Pull request by crwr45**

### CODE REVIEW

File: pkg/config/certified_request_test.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
+// Copyright (C) 2020-2021 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/config/certified_request_test.go:1]: Just `2021` for this one.

File: test-network-function/operator/suite.go [hunk 2]
```diff
@@ -71,26 +70,28 @@ var _ = ginkgo.Describe(testSpecName, func() {
...
 
 func itRunsTestsOnOperator() {
-	operatorsInTest = getConfig()
-	gomega.Expect(err).To(gomega.BeNil())
-	gomega.Expect(operatorsInTest).ToNot(gomega.BeNil())
+	operatorsToQuery, operatorsInTest := getConfig()
+	gomega.Expect(operatorsToQuery).ToNot(gomega.BeNil())
 	certAPIClient := api.NewHTTPClient()
+	for _, certified := range operatorsToQuery {
+		ginkgo.It(fmt.Sprintf("test for Operator Certification Status of %s/%s", certified.Organization, certified.Name), func() {
```
Comment (ryandgoulding) [inline test-network-function/operator/suite.go:85]: Similar to the other comment.

**Review by ryandgoulding — COMMENTED** [review]
Just a few small changes, otherwise looks good.

File: test-network-function/container/suite.go [hunk 3]
```diff
@@ -64,25 +64,31 @@ var _ = ginkgo.Describe(testSpecName, func() {
...
-		// Test for CNF certificates
+		// Query API for certification status of listed containers
+		ginkgo.When("verifying a container is certified", func() {
+			conf := config.GetConfigInstance()
+			cnfsToQuery := conf.CertifiedContainerInfo
+			gomega.Expect(cnfsToQuery).ToNot(gomega.BeNil())
+			certAPIClient = api.NewHTTPClient()
+			for _, cnfRequestInfo := range cnfsToQuery {
+				cnf := cnfRequestInfo
+				ginkgo.It(fmt.Sprintf("tests for Container Certification Status for %s/%s", cnf.Repository, cnf.Name), func() {
```
[the remark this answers is not in the document]
Comment (crwr45) [reply test-network-function/container/suite.go:75]: I've gone with this to reduce repetition and improve the natural flow between the `When` and the `It`
```
ginkgo.When("getting certification status"), ...
	...
	ginkgo.It(fmt.Sprintf("container %s/%s should eventually be verified as certified", cnf.Repository, cnf.Name), ...
```

File: test-network-function/operator/suite.go (hunk 2 above)
Comment (crwr45) [reply test-network-function/operator/suite.go:85]: It bugs me that the `container` and `operator` specs are structured so differently that there isn't an obvious `When` for this in the operator spec. Standardising them is for another time, though.
```
ginkgo.It(fmt.Sprintf("should eventually be verified as certified (operator %s/%s)", certified.Organization, certified.Name), ...
```

**Pull request merged**